<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">QFrame {
    border: 1px solid black;  /* 设置边框为黑色，边框宽度为1像素 */
    background-color: white; /* 设置背景颜色为白色 */
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="frame" native="true">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>971</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    border: 1px solid black;  /* 设置边框为黑色，边框宽度为1像素 */
    background-color: white; /* 设置背景颜色为白色 */
}</string>
       </property>
       <widget class="QPushButton" name="btn1">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>530</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>展示</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>450</width>
          <height>60</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>按数据类型查看</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox1">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>40</y>
          <width>450</width>
          <height>60</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>按测试次数查看</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>雷达图</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>130</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>折线图</string>
        </property>
       </widget>
       <widget class="QWebEngineView" name="radio_widget" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>441</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWebEngineView" name="line_widget" native="true">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>160</y>
          <width>441</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="queryEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid black;  /* 设置边框为红色，边框宽度为2像素 */
    background-color: white; /* 设置背景颜色为蓝色 */
}</string>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
      <widget class="QPushButton" name="queryBtn">
       <property name="geometry">
        <rect>
         <x>239</x>
         <y>40</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>281</width>
         <height>521</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
